akka-http-core/src/main/resources/reference.conf

```diff
@@ -18,6 +18,15 @@ akka.http {
     # the request, no `Server` header will be rendered at all.
     server-header = akka-http/${akka.http.version}
 
+    # EXPERIMENTAL FEATURE: Bind using HTTP/2
```

Comment:
I'd suggest we have a whole section of `preview` configuration settings and put it there. I'd like people to see that they are enabling preview stuff. We can later deprecate those settings in favor of the real setting in a compatible way.